test/presenters/licences/set-up.presenter.test.js

```diff
@@ -3,10 +3,14 @@
 // Test framework dependencies
 const Lab = require('@hapi/lab')
 const Code = require('@hapi/code')
+const Sinon = require('sinon')
```

Comment:
If we've added Sinon to the test we also need to add an `afterEach()` to restore things. We've found tests can be flakey when we forget to do this.

```javascript
  afterEach(() => {
    Sinon.restore()
  })
```

In this case it would be placed after the first `beforeEach()` around line 87.